### PR TITLE
Trying to get Satori-html to work, attempt #1

### PR DESCRIPTION
**Pull request by nightknighto**
Issue: Trying to get satori-html to work in Node.js, but gives me the ESM require issue (same error as in attempt #1)

Attempt #1:
image

### CODE REVIEW

File: src/satori.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import satori from 'satori'
+import { Resvg } from '@resvg/resvg-js';
+
+// failed
+// import { html } from "satori-html";
+
+// failed 
+const html = (...args: string[]) => {
+  // @ts-ignore
+  import('satori-html').then(({ html }) => html(...args));
```
Comment (jeetiss) [inline src/satori.ts]: should be `return import('satori-html')`

File: src/satori.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+export default async function satoriFunc() {
+  const template = html(`
```
Comment (jeetiss) [inline src/satori.ts]: `await` here

File: src/satori.ts (hunk 1 above)
Comment (nightknighto) [reply src/satori.ts]:
```ts
const html = (...args: string[]) => {
  // @ts-ignore
  return import('satori-html')
}
```
like that?

Comment (nightknighto) [reply src/satori.ts]: Tried all possible combinations `return import('..')` `return import('..').then...` `return import('..')(..args)`, they all give the same issue unless I'm missing something.

Comment (jeetiss) [reply src/satori.ts]: like it was but with return.
`return import('satori-html').then(({ html }) => html(...args));`

Comment (jeetiss) [reply src/satori.ts]: > they all give the same issue unless I'm missing something.

could you provide the issue?

Comment (nightknighto) [reply src/satori.ts]: Same one
image

Comment (jeetiss) [reply src/satori.ts]: `ts-node` compile dynamic import to `require` and it is causing the errors, you should configure ts to keep dynamic imports in the final bundle

Comment (nightknighto) [reply src/satori.ts]: Changing `moduleResolution` to `node16` made your solution work! Thank you very much!

**Pull request merged**